services/wallet/router/router_v2.go

```diff
@@ -1031,6 +1031,12 @@ func (r *Router) checkBalancesForTheBestRoute(ctx context.Context, bestRoute []*
 
 	// check the best route for the required balances
 	for _, path := range bestRoute {
+		if path.ProcessorName == pathprocessor.ProcessorBridgeHopName {
+			if path.TxBonderFees.ToInt().Cmp(path.AmountOut.ToInt()) > 0 {
```

Comment:
@briansztamfater yes we could, was thinking about it, but my intention here was to use the minimum value that can be received, just to be safe, and decided to use `AmountOut`, cause it includes slippage and users won't receive 0 tx on the destination chain (and possibly complain) in that case. That's what I wanted to avoid.

If we use AmountIn, then at the destination chain user may receive a tx with 0 value, but if we use AmountOut as a param for comparison, then on the destination chain it will be always greater than 0 value received.

If you think it's better with AmountIn, I can change it easily?